lib/include/hse_ikvdb/kvs_rparams.h

```diff
@@ -57,10 +57,10 @@ struct kvs_rparams {
     uint64_t cn_cursor_vra;
     bool     cn_cursor_kra;
 
-    uint8_t  cn_mcache_kra_params;
-    uint8_t  cn_mcache_vra_params;
-    uint8_t  cn_mcache_wbt;
-    uint32_t cn_mcache_vmax;
+    uint8_t  cn_mmap_kra_params;
+    uint8_t  cn_mmap_vra_params;
+    uint8_t  cn_mmap_wbt;
+    uint32_t cn_mmap_vmax;
```

Comment:
An upcoming PR will remove KVDB's use of mcache maps.  These parameters should probably be renamed.   Thoughts on new names?  They determine how HSE uses `madvise` on memory mapped mblocks.

Reply:
Would like feedback from SQL on impact of this change on automation scripts.